Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -776,30 +777,25 @@ - (NSDictionary *)getBiddingDataWithPlacementId:(NSString *)placementId {
     return @{@"token": returnedToken};
 }
 
-- (BannerSize)getBannerSize:(ISBannerSize *)size {
-    if ([size.sizeDescription isEqualToString:@"RECTANGLE"]) {
-        return BannerSizeMrec;
+- (VungleAdSize *)getBannerSize:(ISBannerSize *)size {
+    VungleAdSize *adSize = [VungleAdSize VungleAdSizeBannerRegular];
+    if ([size.sizeDescription isEqualToString:@"CUSTOM"]) {
+        adSize = [VungleAdSize VungleAdSizeFromCGSize:CGSizeMake(size.width, size.height)];
+    } else if ([size.sizeDescription isEqualToString:@"BANNER"] || [size.sizeDescription isEqualToString:@"LARGE"]) {
+        adSize = [VungleAdSize VungleAdSizeBannerRegular];
+    } else if ([size.sizeDescription isEqualToString:@"RECTANGLE"]) {
+        adSize = [VungleAdSize VungleAdSizeMREC];
+    } else if ([size.sizeDescription isEqualToString:@"LEADERBOARD"]) {
```

Comment:
Does IS iOS have this `LEADERBOARD` size? https://developers.is.com/ironsource-mobile/ios/banner-integration-ios/#step-2

Reply:
Yes. It's not on the doc, but it exists in code.

```
    static NSString* const kSizeBanner = @"BANNER";
    static NSString* const kSizeLarge = @"LARGE";
    static NSString* const kSizeRectangle = @"RECTANGLE";
    static NSString* const kSizeLeaderboard = @"LEADERBOARD";
    static NSString* const kSizeSmart = @"SMART";
    static NSString* const kSizeCustom = @"CUSTOM";
 ```